src/Chapter3.hs

```diff
@@ -910,6 +1017,22 @@ Implement instances of "Append" for the following types:
 class Append a where
     append :: a -> a -> a
 
+newtype Gold = MkGold Int
+
+instance Append Gold where
+  append :: Gold -> Gold -> Gold
+  append (MkGold x) (MkGold y) = MkGold (x + y)
+
+instance Append [a] where
+  append :: [a] -> [a] -> [a]
+  append xs ys = xs ++ ys
```

Comment:
You can eta-reduce the implementation of `append` for brevity. So you can see how easy it is to create instances :slightly_smiling_face: 
```suggestion
  append = (++)
```